README.md

```diff
@@ -52,21 +52,52 @@ Create a virtualenv if required and install required packages using "pip install
     4. The CI_DailyBuildUpdates.py script when invoked with command line arguments info_type as "detailed" and zone, it will display the builds details in the provided zone.
 
         ```python3 CI_DailyBuildUpdates.py --info_type detailed --zone syd04```
+    
+    5. The CI_DailyBuildUpdates.py script when invoked with command line arguments info_type as "detailed" and --job_type as "pa", it will display the builds details in the given job type.
+
+        Supported value [p','z','pa'] 
+
+        pa - ppc64le architecture auxilliary jobs
+
+        p - ppc64le architecture periodic jobs
+        
+        z - s390x architecture periodic jobs
+         
+         ```python3 CI_DailyBuildUpdates.py --info_type detailed --job_type pa```
 
 
 
 2. **CI_JobHistory.py:** The CI_JobHistory.py is a script which allows user to query a specific information from all builds that ran on the CI system within a given date range.  
     
     ```python3 CI_JobHistory.py```
 
+    ```python3 CI_Jobhistory.py --zone ```     This command line allows user to fetch query based on zone type.
+
+    ```python3 CI_Jobhistory.py --job_type ``` This command line allows user to fetch query based on job type.
+             
+    ```python3 CI_Jobhistory.py --filter ```   This command line allows user to fetch query based on search filter.
+        Specify the filter string to fetch jobs (Example 'heavy build' / libvirt / powervs / upgrade /multi / 4.14 / 4.15 / 4.16 / 4.17/ 4.18 )'
+
+
+
     1. Interactive Execution: The CI_JobHistory.py can be executed in a interactive mode by setting JENKINS variable as False in config.ini file.
 
     2. Non-Interactive Execution: The CI_JobHistory.py can be executed in a non-interactive mode by setting JENKINS variable as True in config.ini file, along with the JENKINS variable user needs to provide values for the following variables:
         ```
         selected_ci: CI's from where to fetch the jobs.
+        query_option: Query code to fetch information from builds.
+            •	Check Node Crash: Detects if a node crashed during execution.
+	        •	Brief Job Information: Provides a summary of job execution.
+	        •	Detailed Job Information: Fetches in-depth details of jobs.
+	        •	Failed Test Cases: Lists test cases that failed in the CI run.
```

Comment:
`Failed Test Cases: Provides a list of test cases that are failed in the CI run.`